lib/timers.js

```diff
@@ -409,6 +393,26 @@ function processImmediate() {
   }
 }
 
+function tryOnImmediate(immediate, queue) {
```

Comment:
Hah, beat me to it I see. :P

I was going to leave this until after #4007, in part so that it would look similar.

I suggest we keep the naming consistent in some way with https://github.com/nodejs/node/pull/4007/files#diff-0a5d4868b2b9b17cf9e2c11f1bd1311eR177